src/components/Filter/Filter.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../assets/css/colors';
+
+$color-green-dark: #265732;
+$color-gray-light: #dadada;
+
+.filterHeading {
+  background-color: $color-green-light !important;
+  color: $color-green-dark !important;
```

Comment:
Enig i at det vil være bedre å utvide enn å overskrive bootstrap. Lage nye for lærer og elev virker som en bra løsning.